acceptance/api/v1/application_restart_test.go

```diff
@@ -0,0 +1,88 @@
+package v1_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/epinio/epinio/acceptance/helpers/catalog"
+	"github.com/epinio/epinio/helpers"
+	api "github.com/epinio/epinio/internal/api/v1"
+	v1 "github.com/epinio/epinio/internal/api/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("AppRestart Endpoint", func() {
+	var (
+		namespace string
+	)
+	containerImageURL := "splatform/sample-app"
+
+	BeforeEach(func() {
+		namespace = catalog.NewNamespaceName()
+		env.SetupAndTargetNamespace(namespace)
+	})
+	AfterEach(func() {
+		env.DeleteNamespace(namespace)
+	})
+
+	It("restarts the app", func() {
+		app1 := catalog.NewAppName()
+		env.MakeContainerImageApp(app1, 1, containerImageURL)
+		defer env.DeleteApp(app1)
```

Comment:
Should go into an `AfterEach` block. As not all tests (2 of 3) use that the (two) relevant tests have to go into a `Context` block.

Reply:
Actually, the app creation could (should) then move into a `BeforeEach` block for that `Context` also.